Source/PlasticSourceControl/Private/PlasticSourceControlMenu.cpp

```diff
@@ -629,6 +643,33 @@ void FPlasticSourceControlMenu::AddMenuExtension(FToolMenuSection& Menu)
 			FCanExecuteAction()
 		)
 	);
+
+	FPlasticSourceControlProvider& Provider = FPlasticSourceControlModule::Get().GetProvider();
+	const FString& serverUrl = Provider.GetServerUrl();
+
+	if (serverUrl.Contains("@cloud"))
```

Comment:
I would suggest to make only one call to serverUrl.Find() as to keep only one instance of "@cloud" and then use it both in the if () and then in the Left()

Reply:
Even more minor but at least interesting to share, there is an INDEX_NONE define instead of using -1
eg:

```
			const int32 MergeFromIndex = MergeProgressContent.Find(MergeFromString, ESearchCase::CaseSensitive);
			if (MergeFromIndex > INDEX_NONE)
```